config/environments/production.rb

```diff
@@ -78,4 +78,21 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    domain: "https://tiy-spacebook.herokuapp.com/",
+    authentication: "plain",
+    enable_starttls_auto: true,
+    user_name: ENV["GMAIL_USERNAME"],
+    password: ENV["GMAIL_PASSWORD"],
+  }
+
+  config.action_mailer.default_url_options = { host: "localhost:3000" }
```

Comment:
This should be updated to be the heroku url as well, instead of `localhost:3000`